Generics_Challenge.playground/Contents.swift

```diff
@@ -0,0 +1,59 @@
+import Foundation
+
+struct CountedSet<Element: Hashable>: ExpressibleByArrayLiteral{
+    
+    init(arrayLiteral elements: Element...) {
+        for element in elements {
+            insert(element)
+        }
+    }
+    
+    typealias ArrayLiteralElement = Element
```

Comment:
No need for this line of code since you are not using the new name `ArrayLiteralElement`